app/assets/javascripts/spotlight/blocks/solr_documents_block.js

```diff
@@ -49,20 +49,55 @@ SirTrevor.Blocks.SolrDocuments = (function(){
       '</div>',
     ].join("\n") },
 
+    _itemPanelIiifFields: function(index, data) {
+      return [
+        // '<input type="hidden" name="item[' + index + '][iiif_region]" value="' + (data.iiif_region) + '"/>',
+        '<input type="hidden" name="item[' + index + '][iiif_tilesource]" value="' + (data.iiif_tilesource) + '"/>',
+        '<input type="hidden" name="item[' + index + '][iiif_manifest_url]" value="' + (data.iiif_manifest_url) + '"/>',
+        '<input type="hidden" name="item[' + index + '][iiif_canvas_id]" value="' + (data.iiif_canvas_id) + '"/>',
+        '<input type="hidden" name="item[' + index + '][iiif_image_id]" value="' + (data.iiif_image_id) + '"/>',
+      ].join("\n");
+    },
+    setIiifFields: function(data) {
+      $(this.inner).find('[name$="[iiif_image_id]"]').val(data.imageId);
+      $(this.inner).find('[name$="[iiif_tilesource]"]').val(data.tilesource);
+      $(this.inner).find('[name$="[iiif_manifest_url]"]').val(data.manifest);
+      $(this.inner).find('[name$="[iiif_canvas_id]"]').val(data.canvasId);
+    },
     afterPanelRender: function(data, panel) {
       var context = this;
+      var manifestUrl = data.iiif_manifest || data.iiif_manifest_url;
 
-      if (_.isUndefined(data['image_versions'])) {
-        $.getJSON(this.autocomplete_url().replace("%QUERY", "id:" + data.id), function(data) {
-          var doc = context.transform_autocomplete_results(data)[0];
+      $.ajax(manifestUrl).success(
```

Comment:
This manifest parsing is almost entirely copied from https://github.com/projectblacklight/spotlight/blob/master/app/assets/javascripts/spotlight/search_typeahead.js#L68